versioned_docs/version-24.04/installation/installation-of-a-central-server/using-packages.md

```diff
@@ -322,6 +322,8 @@ apt update && apt install lsb-release ca-certificates apt-transport-https softwa
 
 #### Add Sury APT repository for PHP 8.1
 
+>This step is only necessary for Debian 11, if you are using Debian 12, move on to the next step.
+
```

Comment:
Centreon 24.04 and packages require PHP 8.1 and installation of 8.2 is not automatic, so Centreon packages are not required for this version.
It's not OK actually.